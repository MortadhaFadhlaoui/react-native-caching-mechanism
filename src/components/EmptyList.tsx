import React from "react";
import { StyleSheet } from "react-native";
import { Text } from "react-native-paper";
import { BLUE_SKY } from "../styles/colors";

const EmptyList = () => {
  return <Text style={styles.text}>No Data available</Text>;
};

const styles = StyleSheet.create({
  text: { color: BLUE_SKY, textTransform: "capitalize" },
});
export default EmptyList;
