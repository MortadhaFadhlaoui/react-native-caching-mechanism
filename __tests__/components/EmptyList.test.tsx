import renderer from "react-test-renderer";
import React from "react";
import EmptyList from "../../src/components/EmptyList";

describe("EmptyList", () => {
  it("should show new EmptyList", () => {
    const tree = renderer.create(<EmptyList />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
