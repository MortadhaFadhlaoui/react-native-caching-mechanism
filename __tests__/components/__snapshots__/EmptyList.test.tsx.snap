// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyList should show new EmptyList 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000000",
        "fontFamily": "System",
        "fontWeight": "400",
      },
      Object {
        "textAlign": "left",
      },
      Object {
        "color": "#87CEEB",
        "textTransform": "capitalize",
      },
    ]
  }
>
  No Data available
</Text>
`;
